### PR TITLE
Retry e2e travis job once upon failure

### PR DESCRIPTION
**Pull request by webteckie**
## Description of changes
Sometimes SauceLabs fails due to their own reasons.  This change ensures that we retry any failed e2e test job, at least, once.

## Related issues (if any)
N/A

## Testing

- [X] Please confirm `npm run test-all` ran successfully.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,32 +11,32 @@ matrix:
   include:
     - node_js: '6'
       script:
-        - npm run test-e2e-saucelabs-group
+        - travis_retry npm run test-e2e-saucelabs-group
```
Comment (bassjacob) [inline .travis.yml]: https://docs.travis-ci.com/user/common-build-problems/#travis_retry

**Comment by bassjacob** [comment]
LGTM. noting this will retry a job up to 3 times.

**Comment by jstockwin** [comment]
@webteckie Nice find on this feature!

**Comment by bassjacob** [comment]
and I've seen it in practice on a few failing PR's. this looks awesome so far!

**Pull request merged**